docs/src/examples/components/ItemLayout/index.tsx

```diff
@@ -1,14 +1,15 @@
 import * as React from 'react'
+import { Alert } from '@stardust-ui/react/'
+
 import Types from './Types'
 import Content from './Content'
-import Variations from './Variations'
 import Rtl from './Rtl'
 
 const ItemLayoutExamples = () => (
   <div>
+    <Alert warning>This component is deprecated</Alert>
```

Comment:
Let's try and be as helpful as we can.  Perhaps add link to Layout guide and suggest flex as alternative?